volttron/platform/auth.py

```diff
@@ -892,7 +892,7 @@ def _read(self):
             with open(self.auth_file) as fil:
                 # Use gevent FileObject to avoid blocking the thread
                 before_strip_comments = FileObject(fil, close=False).read()
-                data = strip_comments(before_strip_comments.decode("utf-8"))
+                data = strip_comments(before_strip_comments)
```

Comment:
Can you add this before line#895. This line has been there since volttron-7-preview. Not sure what has changed since last week. 

if isinstance(before_strip_comments, bytes):
    before_strip_comments.decode("utf-8")